game_environment/playing_utils/level_playing_utils.py

```diff
@@ -505,5 +509,12 @@ def step(self, current_actions_map:dict) -> dict[int, list[str]] | None:
             pygame.display.update()
             self.clock.tick(self.fps)
 
+            # Update the time: One hour per step
+            self.time += datetime.timedelta(hours=1)
```

Comment:
Muy interesante el manejo del tiempo en el juego, estoy de acuerdo conque cada step puede equivaler a una hora in game

Reply:
Probamos y si acaso lo podemos cambiar. Traté de usar el pygame.time.Clock, pero luego de varios intentos me di cuenta que eso es mas para manejar el tiempo del display del juego, y que no se ajustaba a lo que necesitabamos porque podríamos tener el tiempo real transcurrido pero eso no sería de tanta utilidad. Mejor que nosotros tengamos el control de ese tiempo.